pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -106,36 +98,45 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 
 			newRule := apimodels.Rule{
 				Name:           rule.Title,
-				Labels:         nil,  // TODO: NG AlertRule does not have labels but does have annotations
+				Labels:         rule.Labels,
 				Health:         "ok", // TODO: update this in the future when error and noData states are being evaluated and set
 				Type:           apiv1.RuleTypeAlerting,
-				LastEvaluation: time.Time{}, // TODO: set this to be rule evaluation time once it is being set
-				EvaluationTime: 0,           // TODO: set this once we are saving it or adding it to evaluation results
+				LastEvaluation: time.Time{},
```

Comment:
Could we implement this too? I believe this would just be the start or end date we use to figure out the evaluation duration.

Reply:
This should be the same as the `EvaluationTime` time for the individual alerts, since they're processed at the same time before being handed off to the state tracker. So we're effectively duplicating the data, unless there's something else we should use to set it.